CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
```

Comment:
![image](https://github.com/sovity/edc-ui/assets/30114507/b9b0b59c-dcea-4c26-a56a-dffbc5ef5f61)

Placeholders are missing, we have them in the default section